apps/chat/src/store/publication/publication.epics.ts

```diff
@@ -128,6 +132,24 @@ const publishEpic: AppEpic = (action$) =>
         return fileBucket !== userBucket;
       });
 
+      const resources = payload.resources.map((resource) => {
+        if (
+          allPublicFileIds.includes(resource.targetUrl) &&
```

Comment:
Need to use `ADD_IF_ABSENT` instead of generate new name
See
https://github.com/epam/ai-dial-core/issues/554
https://github.com/epam/ai-dial-core/pull/555/files